kong/templates/nginx_kong.lua

```diff
@@ -84,7 +84,7 @@ server {
     access_log logs/access.log;
 
 > if ssl then
-    listen ${{PROXY_LISTEN_SSL}} ssl;
+    listen ${{PROXY_LISTEN_SSL}} ssl http2;
```

Comment:
@p0pr0ck5 what would be the use-case? This is between Client <> Kong, not between Kong <> API.